app/ui/src/Pages/Project/pages/KG/components/Filters/Filters.tsx

```diff
@@ -1,13 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styles from './Filters.module.scss';
+import ScoreFilter from './components/ScoreFilter/ScoreFilter';
 
 type Props = {};
 function Filters({}: Props) {
+  const max = 100;
+  const [scores, setScores] = useState<number[]>([20, 80]);
```

Comment:
It would be better to type this state as [number, number], as we are not expecting 1 or  more than 2 numbers.
```suggestion
  const [scores, setScores] = useState<[number, number]>([20, 80]);
```